test/e2e/tls_test.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/vmware-tanzu/antrea/pkg/apis"
+)
+
+const (
+	cipherSuite    = tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 // a TLS1.2 Cipher Suite
+	cipherSuiteStr = "ECDHE-RSA-AES128-GCM-SHA256"
+)
+
+var (
+	cipherSuites             = []uint16{cipherSuite}
+	opensslTLS13CipherSuites = []string{"TLS_AES_128_GCM_SHA256", "TLS_AES_256_GCM_SHA384", "TLS_CHACHA20_POLY1305_SHA256"}
+)
+
+// TestTLSCipherSuites tests Cipher Suite and TLSVersion config on Antrea Apiserver, Controller side or Agent side.
```

Comment:
I don't think we usually capitalize "apiserver"

Reply:
Used "apiserver"